docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.9"
+
+services:
+  app:
+    build:
+      context: .
+    ports:
+      - "8000:8000"
+    env_file:
+      - .env
+    depends_on:
+      - db
+    volumes:
+      - ./:/app
```

Comment:
Mounting the entire current directory into the container can lead to potential issues, such as overwriting files in the container or including unnecessary files. Consider specifying only the necessary directories or files to be mounted.